markdown.html.markdown

```diff
@@ -153,12 +153,11 @@ inside your code -->
 John didn't even know what the `go_to()` function did!
 
 <!-- In Github Flavored Markdown, you can use a special syntax for code -->
-
-\`\`\`ruby <!-- except remove those backslashes when you do this, just ```ruby ! -->
-def foobar
-    puts "Hello world!"
-end
-\`\`\` <!-- here too, no backslashes, just ``` -->
+    ```ruby
```

Comment:
@levibostian If you remove the whitespace, it marks an end to the code block started at the start of the block marked "markdown" which is why I believe "daturkel" introduced the hacky backslashes in this commit: https://github.com/awalGarg/learnxinyminutes-docs/commit/55ded27c4f755c36e1ec09d681e76a7df418e6a9#diff-5744dd7bd81f21547f768566a7e0bc34R135. But markdown (http://spec.commonmark.org/0.22/#indented-code-blocks) allows us to indent code with spaces like this so we can give up the backslashes for a cleaner solution :)